File: 1.FDP/2° Competencia/C++/Basicos/Ejercicio6.cpp
#include <iostream>
using namespace std;

int main(){
	cout << "Convertir de Grados fahrenhed a celsius.\n";
	double Fahrenhed, Celsius;
	cout << "Grados fahrenhed: ";
	cin >> Fahrenhed;
	Celsius= (Fahrenhed - 32) * 5/9;
	cout << "Celsius = " << Celsius << "\n";
	return 0;
}

File: 1.FDP/2° Competencia/Java/Basicos/Ejercicio6.java
import java.util.Scanner;	

public class Ejercicio6 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		//
		System.out.println("Calcular la convercion de grados fahrenhed.\n");
		double Fahrenhed, Convercion;
		System.out.println("Fahrenhed: ");
		Fahrenhed= scan.nextDouble();
		Convercion= (Fahrenhed - 32) * 5/9;
		System.out.println("Celsius= " + Convercion);
		//
	}
}